Src/WitsmlExplorer.Frontend/components/ContentViews/RigsListView.tsx

```diff
@@ -13,15 +13,44 @@ export const RigsListView = (): React.ReactElement => {
       setRigs(selectedWellbore.rigs);
     }
   }, []);
+  const getTableData = () => {
+    return rigs.map((rig) => {
+      return {
+        ...rig,
+        id: rig.uid,
+        ratingDrillDepth: `${rig.ratingDrillDepth?.value ?? ""} ${rig.ratingDrillDepth?.uom ?? ""}`,
+        ratingWaterDepth: `${rig.ratingWaterDepth?.value ?? ""} ${rig.ratingWaterDepth?.uom ?? ""}`,
+        airGap: `${rig.airGap?.value ?? ""} ${rig.airGap?.uom ?? ""}`,
+        rig: rig
+      };
+    });
+  };
 
   const columns: ContentTableColumn[] = [
     { property: "name", label: "Name", type: ContentType.String },
-    { property: "airGap", label: "AirGap", type: ContentType.Number },
+    { property: "uid", label: "Uid", type: ContentType.String },
     { property: "owner", label: "Owner", type: ContentType.String },
-    { property: "typeRig", label: "TypeRig", type: ContentType.String }
+    { property: "typeRig", label: "TypeRig", type: ContentType.String },
+    { property: "manufacturer", label: "Manufacturer", type: ContentType.String },
+    { property: "yearEntService", label: "YearEntService", type: ContentType.String },
+    { property: "classRig", label: "ClassRig", type: ContentType.String },
+    { property: "approvals", label: "Approvals", type: ContentType.String },
+    { property: "registration", label: "Registration", type: ContentType.String },
+    { property: "telNumber", label: "TelNumber", type: ContentType.String },
+    { property: "faxNumber", label: "FaxNumber", type: ContentType.String },
+    { property: "emailAddress", label: "EmailAddress", type: ContentType.String },
+    { property: "nameContact", label: "NameContact", type: ContentType.String },
+    { property: "ratingDrillDepth", label: "RatingDrillDepth", type: ContentType.String },
+    { property: "ratingWaterDepth", label: "RatingWaterDepth", type: ContentType.String },
+    { property: "isOffshoreText", label: "IsOffshore", type: ContentType.String },
+    { property: "airGap", label: "AirGap", type: ContentType.String },
+    { property: "dTimStartOp", label: "DateTimeStartOperating", type: ContentType.DateTime },
+    { property: "dTimEndOp", label: "DateTimeEndOperating", type: ContentType.DateTime },
+    { property: "dTimCreation", label: "DTimCreation", type: ContentType.String },
```

Comment:
I think dTimCreation and dTimLastChange can be removed form the content view given that they are not mentioned in the issue

Reply:
I believe it has to be as it is, else it will be troublesome to load the xml document properly